src/heitech.zer0mqXt.core/Patterns/PubSub.cs

```diff
@@ -1,15 +1,17 @@
 using System;
+using System.Runtime.CompilerServices;
 using System.Threading;
 using System.Threading.Tasks;
 using heitech.zer0mqXt.core.infrastructure;
 using heitech.zer0mqXt.core.transport;
 using NetMQ;
 using NetMQ.Sockets;
 
+[assembly: InternalsVisibleTo("heitech.zer0mqXt.core.tests")]
```

Comment:
About InternalsVisibleTo, see also my comment in `src/heitech.zer0mqXt.core/Infrastructure/SocketConfiguration.cs`.